kube-client/src/client/auth/mod.rs

```diff
@@ -360,9 +360,15 @@ fn token_from_gcp_provider(provider: &AuthProviderConfig) -> Result<ProviderToke
     // Command-based token source
     if let Some(cmd) = provider.config.get("cmd-path") {
         let params = provider.config.get("cmd-args").cloned().unwrap_or_default();
-
+        // This does currently not exist upstream and cannot be specified on disk.
```

Comment:
```suggestion
        // NB: This property does currently not have a counter-part in client-go
        // See https://github.com/kube-rs/kube/issues/1060
```

It's not actually true that it cannot be specified on the disk because the config is just an arbitrary map.